js/client-chat-tournament.js

```diff
@@ -277,6 +277,7 @@
 						if (!this.info.isJoined)
 							this.toggleBoxVisibility(false);
 						this.room.$chat.append("<div class=\"notice tournament-message-start\">The tournament has started!</div>");
+						if (this.info.format.substr(0, 4) === 'gen5' && !Tools.loadedSpriteData['bw']) Tools.loadSpriteData('bw');
 						break;
 
```

Comment:
The problem with loading the data on joined room is that it takes longer for that data to load than it takes for the room to be made, so unless the room is forced to wait, that would only display properly at the second game.